docker-compose.override.yml

```diff
@@ -0,0 +1,7 @@
+# this file is for devs only
```

Comment:
Was this file supposed to be added to source control? It's in the gitignore, which confused me.

Reply:
I wanted it to be in the repo so developers would have it, but also in the `.gitignore` so that when they change it locally, they wouldn't accidentally push it.

But I'm not even sure that it's working lol. I see two options:
1. Remove it from `.gitignore`
2. Remove the override file from the repo

I'm leaning towards 1, wdyt?

Reply:
I'll just go with option 1, we can change it as needed in the future

Reply:
I feel like what can happen then is that
* I change the override locally
* I don't want to commit it
* I add it to gitignore, which undoes option 1

So I think option 2 + some documentation/examples on how to override would be better.